src/pmpo_fortran.f90

```diff
@@ -285,6 +263,28 @@ subroutine polympo_setMeshElm2ElmConn(mpMesh, maxEdges, nCells, cellsOnCell) &
     type(c_ptr), intent(in), value :: cellsOnCell
   end subroutine
   !---------------------------------------------------------------------------
+  !> @brief get the number of elements from the mesh holding by polyMPO
+  !> @param mpMesh(in) mpMesh object
+  !> @param numVtxs(return)) the number of elements
```

Comment:
I prefer `out` to `return` for the annotation of `numVtxs` here since the value of the `numVtxs` variable that is passed in is not used; it is overwritten without being read.  If it was read and then written to then I think `in/out` would be more appropriate.